README.md

```diff
@@ -625,6 +733,14 @@ Future<void>.delayed(Duration.zero, () {
 });
 ```
 
+## Support, feedback and troubleshooting
+
+Please visit <http://support.ably.com/> for access to our knowledge base and to ask for any assistance.
+
+You can also view the [community reported Github issues](https://github.com/ably/ably-flutter/issues).
```

Comment:
Another one inherited from `ably-java`, but my preference is to simply remove this paragraph entirely. It adds little value, is actually slightly misleading and ultimately tells visitors how to 'suck eggs'.

Reply:
Also fixed in https://github.com/ably/ably-flutter/pull/413/commits/112d5c75a6babeb050304a3ed21b2bddd4773b4f